src/global.json

```diff
@@ -3,7 +3,7 @@
     "wrap"
   ],
   "sdk": {
-    "version": "1.0.0-rc1-update1"
+    "version": "1.0.0-preview1-002702"
```

Comment:
The assemblies are RC2, but the SDK + tooling are preview 1. By the time it hits 1.0 the SDK + tooling will be preview 2, and version 1.0 of SDK + tooling is going to be released latter.
This is because DNX was completely deprecated, and the new CLI took over, but since it is completely new, it still isn't as stable to be called a Release Candidate.